wgsl/index.bs

```diff
@@ -1292,6 +1293,25 @@ const_expr
   </xmp>
 </div>
 
+## Return statement ## {#return-statement}
+
+<pre>
+return_stmt
+  : RETURN logical_or_expression?
+</pre>
+
+A `return` statement ends execution of the current function.
+If the function is an entry point, then the current shader invocation
+is terminated.
+Otherwise, evaluation continues with the next expression or statement after
+the evaluation of the call site of the current function invocation.
+
+If the return type of the function is the void type, then the return statement
+must not have an expression.
```

Comment:
This is a fine choice, though C++ (at least) does allow `return foo();` for `void foo();`.

Reply:
I didn't know that!   I'm kind of glad I didn't. :-)